test/asdf_elvish.bats

```diff
@@ -3,12 +3,15 @@
 load test_helpers
 
 setup() {
+  # shellcheck disable=SC1007
   export XDG_CONFIG_HOME= XDG_DATA_HOME= XDG_DATA_DIRS=
```

Comment:
Though this make sense if you read the Shellcheck report for SC1007, it is easier for devs who don't know SC1007 and for all future contributors that we just change this code to:

```
  export XDG_CONFIG_HOME=
  export XDG_DATA_HOME=
  export XDG_DATA_DIRS=
```

Otherwise, we must mentally keep track of why a Shellcheck rule has been turned off. If I come back to this code 1yr from now, I don't want to have to look up the rule to be reminded of it, re-read the code to ensure it is being used correctly and decide that disabling the rule is the correct thing to do. All that for a minor convenience in syntax.

It also makes refactoring easier as we don't need to make this evaluation of whether the rule should still be disabled or not.

Generally, the preference would be to just change the code than disable a rule. It is more future proof.

Sorry for the nit.